### ISSUE TITLE
Controller set to "None" when going from one core to another with different device type

### ISSUE THREAD

**Issue by fr500** [libretro/RetroArch#4410]
**Symptom:**
After changing cores for example from Snes9x to Glupen64 controller type is set to "None"

**Expected behavior:**
If the controller is not the same it should default to the first controller (in Glupen64's case it's called "Controller")

**Comment by fr500** [comment]
It doesn't seem to happen every single time...

**Comment by twinaphex** [comment]
Does this still happen?

**Comment by fr500** [comment]
Yes, actually on a first installation on a shield controller was set to None according to @webgeek1234
